src/reducers/tickets/badge-print-reducer.js

```diff
@@ -40,8 +39,8 @@ const badgePrintReducer = (state = DEFAULT_STATE, action) => {
             const { data, current_page, last_page, total } = payload.response;
 
             const prints = data.map(p => {
-                const created = p.created ? epochToMoment(p.created).format('MMMM Do YYYY, h:mm:ss a') : '';
-                const print_date = p.print_date ? epochToMoment(p.print_date).format('MMMM Do YYYY, h:mm:ss a') : '';
+                const created = p.created ? p.created : 'N/A';
+                const print_date = p.print_date ? p.print_date : 'N/A';
```

Comment:
do here
 epochToMomentTimeZone(t.created, currentSummit.time_zone_id).format('MMMM Do YYYY, h:mm:ss a')},
if not null